homework-4/home-work-4.py

```diff
@@ -0,0 +1,10 @@
+var = input("Уведіть текст:")
+if str.isdigit(var) == True:
```

Comment:
`str` methods work for values as well. So you can call this method for a value you want to check. In your case, you can do it like this: `if var.isdigit() == True:`.

Also, you don't need to do extra checks if some condition is True since, behind the scene, Python will do it for you. So, the line can be simplified to `if var.isdigit():`